doc/jcli/rest.md

```diff
@@ -206,3 +206,34 @@ value: 990
 * `delegation` is the Stake Pool Identifier the account is delegating to.
   it is possible this value is not set if there is no delegation certificate
   sent associated to this account.
+
+
+## Node settings
+
+Fetches node settings
+
+```
+jcli rest v0 node settings get <options>
```

Comment:
shouldn't have `node` in the command, also `jcli rest v0 settings get --output-format yaml -h <host>` doesn't work.

Reply:
Good one about the docs.

But I can't reproduce JCLI not working:
```
./jcli rest v0 settings get --output-format yaml -h http://127.0.0.1:8443/api
---
block0Hash: 8d94ecfcc9a566f492e6335858db645691f628b012bed4ac2b1338b5690355a7
block0Time: "2019-07-09T12:32:51+00:00"
consensusVersion: bft
currSlotStartTime: ~
fees:
  certificate: 4
  coefficient: 1
  constant: 2
maxTxsPerBlock: 100
```
What's the failure on your computer?

Reply:
Yeah I edited it out because I forgot to append the `/api` :-) sorry about the noise.

Reply:
Whew, good to hear :)